index.js

```diff
@@ -41,10 +41,26 @@ function configureFiles(files, metalsmith, done) {
       var file = files[fname];
       file.ext = ext;
       file.root = '/';
-      file.template = 'page.hbt';
       file.path = makePath(fname, ext);
       file.title = makeTitle(fname, ext);
-      file.nav_list = file.path !== 'index.html';
+      if (file.path === 'index.html') {
+        file.template = 'main.hbt';
+        repos = ['Arrays', 'Boxengine', 'Boxpanel', 'Disposable',
```

Comment:
can you make these `CapCase` where it makes sense: `SplitPanel`, `StackedPanel`, etc..